docs/application/dotnet/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,142 @@
+
+# Component Manager
+
+Component manager provides information about installed and running components.
+
+The main features of the `Tizen.Applications.ComponentBased.ComponentManager` class include:
+
+-   Managing running component context
+
+    For the running components, you can retrieve the component running context and operate on it. You can [manage thec component running context](#manage_context) with the `Tizen.Applications.ComponentBased.ComponentManager` class.
```

Comment:
Please correct the spelling mistake "thec":

You can [manage the component running context](#manage_context) with the ......

Reply:
Done